### ISSUE TITLE
avoid minecraft server timeout when players disconnect

### ISSUE THREAD

**Issue by gekigek99** [gekigek99/minecraft-server-hibernation#63]
@SmirkVG from issue #45

> When players disconnect from the server, as far as the Minecraft server console is concerned, they time out, not disconnect. I understand that this is probably a product of your script serving as a "middle man" of sorts between players and the server, but is it possible, perhaps in a future version, for the script to tell the Minecraft server console that a player disconnected the moment they do, so that they don't have to "time out"?

when a player disconnects, msh should actively disconnect from the server (to immediately trigger the "player disconnected from server", without having to wait few seconds)

**Comment by Br31zh** [comment]
I dreaded that day. :laughing: 

```
[22:05:20] [Server thread/INFO]: <Player A> Goodbye
[22:05:27] [Server thread/INFO]: <Player B> same
22:05:34 *** A PLAYER LEFT THE SERVER! - 1 players online ######## (this is player A that have really quit)
[22:05:51] [Server thread/INFO]: Player A was slain by Spider
[22:05:58] [Server thread/INFO]: Player A lost connection: Timed out
[22:05:58] [Server thread/INFO]: Player A left the game
```

**Comment by gekigek99** [comment]
sorry I don't get it ahah... what does this log mean?

**Comment by Br31zh** [comment]
The Player A have exited his client at 22:05:34. But, since msh don’t really disconnected him, he was killed by a spider at 22:05:51, and only then he timed out at 22:05:58.

So he’s dead after he disconnected because of this issue.

**Comment by gekigek99** [comment]
oh wow... thanks for highlighting this... I've never thought about it

will fix as soon as possible

**Comment by gekigek99** [comment]
solved in 8e994c005007b53e265a02bc44723edff089f165

**Comment by Br31zh** [comment]
Thanks, that works perfectly.
